package.json

```diff
@@ -27,5 +27,8 @@
     "istanbul": "^0.4.5",
     "jasmine-node": "^1.14.5",
     "webpack": "^2.1.0-beta.27"
+  },
+  "dependencies": {
+    "angular": "1.5.6"
```

Comment:
I think we still need the `^`. I believe our component should work with 1.5.6 *and above*.